source/reference/minio-server/settings/root-credentials.rst

```diff
@@ -13,6 +13,8 @@ Root Access Settings
 This page covers settings that control root (superuser) access for the MinIO process. 
 The root user has complete access and permissions to perform operations on the MinIO deployment.
 
+Root User and Root Password are required even if you use the :kes-docs:`MinIO Key Encryption Service <>` or other key management utility.
```

Comment:
:thinking: I wonder where this comes from though. I don't think we imply that enabling KES services would offload root IAM. 

We can keep it because it harms nothing but we might need to look at our SSE docs and see if we're making some implications.